process/block/preprocess/rewardsHandler.go

```diff
@@ -436,30 +443,41 @@ func (rtxh *rewardsHandler) verifyCreatedRewardsTxs() error {
 	rtxh.mut.Lock()
 	defer rtxh.mut.Unlock()
 
+	var err error
+	defer func() {
+		if err != nil {
+			rtxh.displayCalculatedRewardsTxs()
```

Comment:
this is called 2 times. already called in createAllMiniblocks for reward. Which is called by validator and leader as well.

Reply:
Yep. Removed from this method